packages/docs/src/routes/(routes)/docs/install/vike/+page.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Install daisyUI for Vike
+desc: How to install Tailwind CSS and daisyUI in a Vike project
+---
+
+### 1. Create a new Vike project
+
+Create a new Vike project in the current directory
+
+```:Terminal
```

Comment:
Please use `npm create vike`. We don't want to introduce a new scaffolding tool because then we have to explain what is which is not the objective of this page.
Also, no need for flags for different frameworks.

Reply:
Done